api/specs/webserver/openapi-services.yaml

```diff
@@ -16,6 +16,20 @@ paths:
                 $ref: "./components/schemas/pipeline.yaml#/components/schemas/PipelineCreatedEnveloped"
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
+
+  /computation/pipeline/{project_id}/stop:
```

Comment:
coording to our standards this should be ``/computaton/pipeline/{project_id}:stop`` :-)

Reply:
hmm that is weird.. cause it is supposed to be.. probably some bad merging stuff

Reply:
ah no it is actually correct see here:
https://github.com/sanderegg/osparc-simcore/blob/6e04b4413c0e090bff5cd49c52a41888caf56cde/api/specs/webserver/openapi.yaml#L142